model-optimizer/extensions/middle/TensorIteratorMerge.py

```diff
@@ -304,7 +303,7 @@ def replace_pattern(graph, match: dict):
 
             if ext_out['axis'] is not None:
                 # Insert unsqueezing resize at output port that has partitioning
-                assert not ext_out['internal_data_id'].has_valid('value')
+                # assert not ext_out['internal_data_id'].has_valid('value')
```

Comment:
Commenting out of this assert enabled conversion one of the models with dynamic dimensions.

Reply:
Okay, then maybe we should remove this line? Or add some comment why this is commented?

Reply:
he assert was removed because previously sub-graph with constant output value from the TI was not supported for some reason. Now it works. It doesn't make sense to keep this line commented, so I have removed it.